context.md

```diff
@@ -226,8 +270,7 @@ instructor.displayFoods();
 
 ### Bind
 
-Another way is to use the bind method on the function to force the function to
-always use the specified value as its context.
+Another way is to use the bind method on the function to force the function to always use the specified value as its context.
```

Comment:
creates a copy of the original function - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/bind